v2/src/Api.tsx

```diff
@@ -67,7 +32,7 @@ export interface ApiDirection {
 
 export interface ApiPath {
   /* [[lon, lat]] */
-  point: [[number, number]] 
+  point: [[number, number]]
```

Comment:
Is this intended to be an array with a single point element?

Reply:
it's supposed to be an array of points, so like `[[lon1,la1], [lon2,lat2], ...]`

Reply:
I think that would have to be `[number, number][]`

Reply:
Ok thanks I wasn't sure